synapse/models/material.py

```diff
@@ -0,0 +1,38 @@
+'''
+A data model focused on material objects.
+'''
+
+def getDataModel():
+    return {
+        'prefix':'mat',
+
+        'types':(
+            ('mat:item',{'subof':'guid','doc':'A GUID assigned to a material object'}),
+            ('mat:spec',{'subof':'guid','doc':'A GUID assigned to a material specification'}),
+            ('mat:specimage',{'subof':'sepr','sep':'/','fields':'item,mat:spec|file,file:bytes'}),
+            ('mat:itemimage',{'subof':'sepr','sep':'/','fields':'item,mat:item|file,file:bytes'}),
+            #TODO add base types for mass / volume
+        ),
+
+        'forms':(
+
+            ('mat:item',{},(
+                ('name',{'ptype':'str:lwr','doc':'The human readable name of the material item'}),
```

Comment:
clarify this represents the physical instance of an item.
one mat:item may contain other mat:item's ?